### DISCUSSION TITLE
Set the interfacte detail in note by default

### DISCUSSION THREAD

**Discussion by tomitomas** [tteck/Proxmox#2116] [Ideas]
Hello, 

the scripts are A-MA-ZING ! 
thanks for all the work done. 

just a suggestion : 
have you ever consider to set in the "note" part the interface url/port to use by default?  
--> you already provide this info for each script in your github page, but that might also be great to get it down in the "note" part after the 'buy me a koffee' link. 
what u think ? 

for ESPHOME : 
`ESPHome Interface: IP:6052`

for Vaultwarden :
Vaultwarden Interface: IP:8000
Vaultwarden Admin Interface: IP:8000/admin

etc ...

**Comment by tomitomas** [comment]
Hi @tteck 

an opinion ?

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/3303
